book/src/filters.md

```diff
@@ -408,19 +408,29 @@ This will output formatted YAML for any value that implements the required
 ## Custom Filters
 [#custom-filters]: #custom-filters
 
-To define your own filters, simply have a module named filters in scope of the context deriving a `Template` impl.
+To define your own filters, simply have a module named `filters` in scope of the context deriving a `Template` impl and define the filters as functions within this module. The functions must have at least one argument and the return type must be `::askama::Result<T>` where `T` is some type that implements `Display`. 
```

Comment:
I'd prefer if you use some hard line wrapping (at about 100 columns).

Also, I think it's not true that a filter must return a `T` that implements `fmt::Display` -- that's what can be used in a template expression, but since filters can be chained any particular filter doesn't have to yield that.

Reply:
Thanks! That is totally true!
Changed to mention that is only needed at the end of a chain of filters.